client/src/components/PortfolioFilters.tsx

```diff
@@ -24,11 +24,11 @@ import MinMaxSelect from "./MinMaxSelect";
 
 import "styles/PortfolioFilters.scss";
 
-type PortfolioFilterProps = {
+type PortfolioFilterProps = withI18nProps & {
   logPortfolioAnalytics: LogPortfolioAnalytics;
 };
 
-export const PortfolioFilters = React.memo(
+const PortfolioFiltersWithoutI18n = React.memo(
```

Comment:
```suggestion
const PortfolioFiltersComponent = React.memo(
```

Reply:
another option 

```suggestion
const BasePortfolioFilters = React.memo(
```

Reply:
Yeah the naming is a little odd, but I was just copying the convention that used elsewhere in WOW - there are a lot of cases like this and they all just keep the final component name with `WithoutI18n` appended.

Reply:
didn't realize it's an existing convention! thanks for clarifying